### DISCUSSION TITLE
Cannot push message to terminal suddenly

### DISCUSSION THREAD

**Discussion by linzx0403** [relayrides/pushy#919] [General]
We use APNS notification to push message to ios devices, it is always normal. But cannot push message to terminal suddenly 。

I use pushy:0.14.2 as api interface, and cannot receive response from CompletableFuture . Also the p12 certificate is valid, not expired.

My question is  how did this happen, ans how to resolve the problem. May I have your suggestions. Thanks.

**Comment by jchambers** [comment]
Hi! I'm really sorry, but I think we may have a little bit of a language barrier here. I'm not quite sure what you mean by "push message to terminal." Can you say more about what you're trying to do, or can you provide an example?
